spring-boot-admin-server-cloud/src/main/java/de/codecentric/boot/admin/server/cloud/discovery/InstanceDiscoveryListener.java

```diff
@@ -66,6 +68,10 @@
      */
     private Set<String> services = new HashSet<>(Collections.singletonList("*"));
 
+    private Map<String, String> instancesMetadata = new HashMap<>();
```

Comment:
Please add a comment to the field so that a description is included to the configuration metadata.